### PR TITLE
Vector: expose ++ to user

### PR DESCRIPTION
**Pull request by liyishuai**
**Kind:** feature
- [x] Entry added in CHANGES.

### CODE REVIEW

**Comment by herbelin** [comment]
This was indeed missing. Well-spotted.

In theory, there could be incompatibilities but the PR is going the expected way and the change is documented. So that's all good to me. Thanks

**Comment by herbelin** [comment]
PS: Assigning it to me, since apparently single reviewer. Shall merge between Thursday late and Friday morning if no other comments by then.

**Comment by Zimmi48** [comment]
Actually @liyishuai had already prepared a fix for cross-crypto and this fix has been merged, so we only have to relaunch the test (I'm doing it).

**Comment by Zimmi48** [comment]
Hum, there are two weird things with CI. First, @coqbot should have overridden the failed status of cross-crypto since it is now a success. Second, this build "skipped" a lot of jobs and I have no idea why: https://gitlab.com/coq/coq/pipelines/26497628 (cc @SkySkimmer @ejgallego).

@liyishuai To be sure that everything is good and that there are no more incompatibilities, a rebase of this PR against the latest master would allow a fresh CI run. Should the entry in CHANGES mention that this is a potential source of incompatibility?

**Comment by SkySkimmer** [comment]
Very strange pipeline indeed, no clue what's going on.

**Comment by herbelin** [comment]
Rescheduled for Monday.

File: CHANGES [hunk 1]
```diff
@@ -134,6 +134,14 @@ Display diffs between proof steps
   details.  Showing diffs in Proof General requires small changes to PG
   (under discussion).
 
+Notations
+
+- Added [++] infix for [VectorDef.append].
```
Comment (herbelin) [inline CHANGES]: Is there any reason to use here `[` ... `]` and not `` ` `` ... `` ` `` like in the rest of the paragraph? 

[Note that there is an ongoing [discussion](https://github.com/coq/coq/issues/8180) on the style to use for  `CHANGES` and, even if w/o final conclusion yet, it seems that we are going towards adopting markdown for `CHANGES`.]

**Review by herbelin — COMMENTED** [review]
Hi, looking again at your changes to `CHANGES`, I realized that I had one minor comment.

File: CHANGES (hunk 1 above)
Comment (liyishuai) [reply CHANGES]: I realized there are two styles after the first commit. Maybe I should unify into ``` ` ``` style.

**Pull request merged**